x-pack/filebeat/module/oracle/database_audit/ingest/pipeline.yml

```diff
@@ -22,13 +22,32 @@ processors:
       field: message
       patterns:
         - "%{GREEDYDATA:tmp_timestamp}\\\nLENGTH : '%{GREEDYDATA:LENGTH}'\\\n(?m)%{GREEDYDATA:audit}"
+        - "%{GREEDYDATA:tmp_timestamp}\\\nLENGTH: \"%{GREEDYDATA:LENGTH}\"\\\n(?m)%{GREEDYDATA:auth}"
   - kv:
       field: audit
       field_split: "\\\n(?=[a-zA-Z])"
       value_split: ":\\S\\d+\\S(?= ')"
       trim_value: " '"
       trim_key: " "
       prefix: oracle.database_audit.
+      if: ctx?.audit != null && ctx?.auth == null
+  - gsub:
+      field: auth
+      pattern: "\"\\s"
+      replacement: "\"\\|"
+      ignore_missing: true
+  - kv:
+      field: auth
+      field_split: "\\|"
```

Comment:
Are we confident that there is never a `|` in fields in `auth`?

Reply:
Yes we are